pages/common/git-log.md

```diff
@@ -3,14 +3,10 @@
 > Show a history of commits.
 > More information: <https://git-scm.com/docs/git-log>.
 
-- Show the sequence of commits starting from the current one, in reverse chronological order:
+- Show the sequence of commits starting from the current one, in reverse chronological order of the git repository in the current working directory:
 
 `git log`
 
-- Show the history of a particular file or directory, including differences:
-
-`git log -p {{path/to/file_or_directory}}`
```

Comment:
Hey @sbrl, I think the `-p` argument is important and really useful to get it handy, what if we keep this example and we merge it with another like `--stat`. Or, option B, what if we remove this complex example:

```bash
git log --oneline --decorate --all --graph
```